src/components/appBar/AppBar.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { Text, View, TouchableOpacity } from 'react-native'
+
+import styles from './style'
+import { Icons } from '../../constants/constants'
+
+class AppBar extends React.Component {
+    onPress(navigation, pageName) {
```

Comment:
onPress = () => {
        const { navigation, pageName } = this.props
        navigation.navigate(pageName)
    }